protocol/avdl/keybase1/kbfs_common.avdl

```diff
@@ -51,4 +51,32 @@ protocol kbfsCommon {
     Folder folder;
     int requestID;
   }
+
+  record FSSyncStatusRequest {
+    int requestID;
+  }
+
+  record FSPathSyncStatus {
+    boolean publicTopLevelFolder;
```

Comment:
I always though this bool was a little awkward... is it needed. Should it be two bools `isPublic`, `isTLF` or an enum for `FolderType`.